packages/twitch-helpers/src/lib/types.ts

```diff
@@ -0,0 +1,185 @@
+import type { TwitchEventSubTypes, TwitchHelixBroadcasterType, TwitchHelixUserType } from './enums';
+
+export interface TwitchHelixResponse<T> {
+	data: T[];
+}
+
+export interface TwitchHelixOauth2Result {
+	access_token: string;
+
+	expires_in: number;
```

Comment:
I know those newlines come from Skyra's code, but please, either document the fields with *multiline* blocks, or drop the newlines.

Style-wise, I'm fine with newlines as long as they're to separate documented fields.

Reply:
They actually come from the TypeScript Interfacer Sorter vscode extension lol